app/client/src/components/pages/Community.Tabs.Monitoring.js

```diff
@@ -1022,93 +1085,56 @@ function MonitoringTab({ monitoringDisplayed, setMonitoringDisplayed }) {
                     return a.key > b.key ? 1 : -1;
                   })}
               </tbody>
-            </table>
-
-            <p>
-              <strong>
-                <em>{watershed}</em> totals:{' '}
-              </strong>
-            </p>
-            <table
-              css={modifiedTableStyles}
-              aria-label="Monitoring Totals"
-              className="table"
-            >
-              <tbody>
+              <tfoot>
                 <tr>
-                  <td>
-                    <em>
-                      <GlossaryTerm term="Monitoring Samples">
-                        Monitoring Samples:
-                      </GlossaryTerm>
-                    </em>
-                  </td>
-                  <td>{Number(totalSamples).toLocaleString()}</td>
+                  <th>Totals</th>
+                  <th>{Number(totalDisplayedLocations).toLocaleString()}</th>
+                  <th>{Number(totalDisplayedSamples).toLocaleString()}</th>
+                  <th>{Number(totalDisplayedMeasurements).toLocaleString()}</th>
                 </tr>
-                <tr>
-                  <td>
-                    <em>
-                      <GlossaryTerm term="Monitoring Measurements">
-                        Monitoring Measurements:
-                      </GlossaryTerm>
-                    </em>
-                  </td>
-                  <td>{Number(totalMeasurements).toLocaleString()}</td>
+                <tr className="download-links">
+                  <th colSpan="4">
+                    <div>
+                      <a
+                        rel="noopener noreferrer"
+                        target="_blank"
+                        data-cy="portal"
+                        href={portalUrl}
+                      >
+                        <i
+                          css={iconStyles}
+                          className="fas fa-filter"
+                          aria-hidden="true"
+                        />
+                      </a>
+                      <span>
+                        Advanced Filtering &nbsp;
+                        <small css={modifiedDisclaimerStyles}>
+                          (opens new browser tab)
+                        </small>
+                      </span>
+                    </div>
+                    <div>
+                      <span>Download All &nbsp;</span>
```

Comment:
```suggestion
                      <span>Download Selected Data for <em>{watershed}</em> &nbsp;</span>
```

Kiki wanted us to change the text to "Download Selected data for {watershed}".

Reply:
Does she actually want the watershed name, or just the word "Watershed"?

Reply:
I was thinking the watershed name, just because that is what we do in most places. I think that will be a little to wordy though, so maybe just try the word "Watershed" for now.